programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -616,26 +641,38 @@ static void on_event(const base_event *ev) {
 
             send_events_message(&svc_event, (to_json_fn) services_event_to_json, true);
             if (svc_event.AddedServices != NULL) {
+#if !defined(_WIN32)
```

Comment:
This pointer needs to be free'd on Windows, since it is allocated unconditionally on all platforms. If you are seeing a crash when free'ing this on Windows then something else is causing the crash and it should be addressed.

Reply:
yes, this is crashing in windows.